system/Common.php

```diff
@@ -21,6 +21,7 @@
 use CodeIgniter\HTTP\Exceptions\HTTPException;
 use CodeIgniter\HTTP\RedirectResponse;
 use CodeIgniter\HTTP\RequestInterface;
+use CodeIgniter\HTTP\Response;
```

Comment:
I know our HTTP interfaces and instances are a total mess. But why was this added? I don't see it in your new methods.

Reply:
Good catch. It is because CSP instance was created in the Response at first.

Removed.